mubench.reviewsite/templates/detector.phtml

```diff
@@ -255,7 +255,7 @@ $columns_to_ignore = array(
                 <td class="<?= $style_classes ?>">
                     <?php if($misuse->hasConclusiveReviewState()) : ?>
                         <?php if($misuse->misuse_tags): ?>
-                            <?php foreach($misuse->misuse_tags as $tag) : ?>
+                            <?php foreach($misuse->misuse_tags->unique('id') as $tag) : ?>
```

Comment:
We should use `getTags()` here and remove the surrounding check for `hasConclusiveReviewState()`. Then i consistently see my own tags during review and all tags after reviews reached a conclusive state.